django_redshift_backend/base.py

```diff
@@ -159,8 +159,10 @@ def create_model(self, model):
                 definition += " %s" % col_type_suffix
             params.extend(extra_params)
             # FK
-            if field.rel and field.db_constraint:
-                to_table = field.rel.to._meta.db_table
+            if field.remote_field and field.db_constraint:
```

Comment:
`Field.rel` is deprecated in 1.9 and removed in 2.0.

https://docs.djangoproject.com/en/2.0/releases/1.9/#field-rel-changes